src/sveta/Assignment_23_CharacterFriquency.java

```diff
@@ -0,0 +1,34 @@
+/*Assignment - 23 : 29th Jan'2024
+Print all the characters from given String having frequency 1 
+(print unique characters from given string) [without indexOf & lastIndexOf]
+
+String str = "technocredits";
+output : hnordis 
+*/
+package sveta;
+
+public class Assignment_23_CharacterFriquency {
+
+	public static void frequencyOfCharacter() {
+		String str="technocredits";
+		
+		char ch=' ';
+		int frequency = 0;
+		for(int i=0;i<=str.length()-1;i++) {
+			ch =  str.charAt(i);
+            if(ch == str.charAt(i)) {
```

Comment:
Improper alignment

Reply:
i can't see code like above comment in file. how i can solve. can you once check my screenshot attached with this pr.my code look like above mention screenshot.